src/commands/scan.yml

```diff
@@ -0,0 +1,64 @@
+description: >
+  Automatically test your REST APIs with your OpenAPI specs and Postman collections
+parameters:
+  mapi-token:
+    type: env_var_name
+    description: Mayhem for API service account token
+    default: MAPI_TOKEN
+  api-url:
+    type: string
+    description: URL pointing to your running API you started before calling this action (don't fuzz your production API!)
+  api-spec:
+    type: string
+    description: Path to your Swagger 2.0 or OpenAPI 3.0 specification, or your Postman collection. Can be a URL or postman collection ID.
+  target:
+    type: string
+    description: The organization-scoped name of your target, such as example/my-test-target. Auto-generated from your repository name if unspecified.
+    default: CIRCLE_PROJECT_REPONAME
+  zap-api-scan:
+    type: string
+    description: Include results from ZAP - API Scan. This run ZAP - API Scan and merge the results with your Mayhem for API results.
+    default: 'false'
+  duration:
+    type: string
+    description: Scan duration
+    default: auto
+  # TODO: sarif + html
+  # sarif-report:
+  #   type: string
+  #   description: SARIF output path. You can upload it to GitHub using the 'github/codeql-action/upload-sarif@v1' action
+  # html-report:
+  #   type: string
+  #   description: HTML output path. You can artifact to GitHub using the 'actions/upload-artifact@v2' action
+  experimental-rewrite-plugin:
```

Comment:
Can probably drop the `experimental-` prefix from this and the below option. I will add a separate ticket to do the same for the `mapi` cli.

Reply:
I will revisit this after the CLI has been updated!